src/views/Completed.vue

```diff
@@ -209,6 +229,148 @@ export default defineComponent({
       });
       return popover.present();
     },
+
+    async fetchShipmentMethods() {
+      const payload = prepareOrderQuery({
+        viewSize: "0",  // passing viewSize as 0, as we don't want to fetch any data
+        queryFields: 'productId productName virtualProductName orderId search_orderIdentifications productSku customerId customerName goodIdentifications',
+        groupBy: 'picklistBinId',
+        sort: 'orderDate asc',
+        defType: "edismax",
+        filters: {
+          picklistItemStatusId: { value: '(PICKITEM_PICKED OR (PICKITEM_COMPLETED AND itemShippedDate: [NOW/DAY TO NOW/DAY+1DAY]))' },
+          '-shipmentMethodTypeId': { value: 'STOREPICKUP' },
+          facilityId: { value: this.currentFacility.facilityId },
+          productStoreId: { value: this.currentEComStore.productStoreId }
+        },
+        facet: {
+          "shipmentMethodFacet": {
+            "excludeTags": "shipmentMethodTypeIdFilter",
+            "field": "shipmentMethodTypeId",
+            "mincount": 1,
+            "limit": -1,
+            "sort": "index",
+            "type": "terms",
+            "facet": {
+              "groups": "unique(orderId)",
+              "itemCount": "sum(itemQuantity)"
+            }
+          }
+        }
+      })
+
+      try {
+        const resp = await UtilService.fetchShipmentMethods(payload)
```

Comment:
Check if we need reset of values

Reply:
The component gets unmounted which automatically reset the values, thus we does not need to handle it.